integration/tests/cook/test_basic.py

```diff
@@ -213,19 +215,34 @@ def test_max_runtime_exceeded(self):
         job_timeout_interval_ms = job_timeout_interval_seconds * 1000
```

Comment:
Should these just be `job_sleep_seconds` and `job_sleep_ms`? I kept on having to scroll up to check what these were actually doing